readme.md

```diff
@@ -671,7 +671,7 @@ The default Got options.
 
 ## Errors
 
-Each error contains (if available) `body`, `statusCode`, `statusMessage`, `host`, `hostname`, `method`, `path`, `protocol`, `url`, and `gotOptions` properties to make debugging easier.
+Each error contains `host`, `hostname`, `method`, `path`, `protocol` and `url` properties to make debugging easier.
```

Comment:
I don't think you meant to delete `gotOptions`.

Reply:
you're right, fixed :D